application/controllers/Vendas.php

```diff
@@ -333,6 +347,14 @@ public function excluirProduto()
             redirect(base_url());
         }
 
+        $editavel = $this->vendas_model->isEditable($this->input->post('idVendas'));
+        if (!$editavel) {
+            return $this->output
+                ->set_content_type('application/json')
+                ->set_status_header(200)
+                ->set_output(json_encode(['result' => false, 'messages'=> '<br /><br /> <strong>Motivo:</strong> Venda já faturada']));
+        }
+
```

Comment:
Não retornar status code 200 quando for erro. Recomendo usar 400 ou 422.

Reply:
corrigido